src/utils.ts

```diff
@@ -98,6 +134,31 @@ export function getRequiredGazeboDistributions(): string[] {
 	return requiredGazeboDistributionsList;
 }
 
+/**
+ * Check the compatability of the Ubuntu version against the
+ * Gazebo distribution. Throws an error if incompatible
+ * combination found
+ *
+ * @param requiredGazeboDistributionsList
+ * @param ubuntuCodename
+ */
+export function checkUbuntuCompatibility(
+	requiredGazeboDistributionsList: string[],
+	ubuntuCodename: string,
+) {
+	requiredGazeboDistributionsList.forEach((element) => {
+		const idx = validGazeboDistroList.findIndex((obj) => {
+			return obj.name === element;
+		});
+		const compatibleUbuntuList: string[] = Object.values(
+			validGazeboDistroList[idx].compatibleUbuntuDistros,
+		);
+		if (compatibleUbuntuList.indexOf(ubuntuCodename) <= -1) {
+			throw new Error("Incompatible Gazebo and Ubuntu combination.");
```

Comment:
Can we include in the error the webpage to look for the valid combinations? Probably https://gazebosim.org/docs/latest/getstarted/#step-1-install ?